README.md

```diff
@@ -1,36 +1,44 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+## About
 
-## Getting Started
+College Predictor provides the list of colleges given :
+
+- Exam
+- Category
+- Category rank
+- Round Number
+- Gender
+- Home State
 
-First, install dependencies (`nextjs-course`) and run the development server:
+It also finds the Scholarships given :
```

Comment:
`finds a list of applicable scholarships given:`

Reply:
resolved